jupyter_sparkmonitor/test_sparkmonitor.py

```diff
@@ -0,0 +1,46 @@
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from absl import flags
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class JupyterTestCase(DataprocTestCase):
+    OPTIONAL_COMPONENTS = 'ANACONDA,JUPYTER'
+    COMPONENT = 'sparkmonitor'
+    INIT_ACTIONS = ['jupyter_sparkmonitor/sparkmonitor.sh']
+
+    def verify_instance(self, name, jupyter_port):
+        verify_cmd_pip_check = "/opt/conda/default/bin/pip list | grep 'sparkmonitor'"
+        self.assert_instance_command(name, verify_cmd_pip_check)
+
+        verify_cmd = "curl {} -L {}:{} | grep 'Jupyter Notebook'".format(
+            "--retry 10 --retry-delay 10 --retry-connrefused", name,
+            jupyter_port)
+        self.assert_instance_command(name, verify_cmd)
+
+
+    @parameterized.parameters(
+        ("SINGLE", ["m"]),
+        ("STANDARD", ["m"]),
+    )
+    def test_sparkmonitor(self, configuration, machine_suffixes):
+        # Use 1.4 version of Dataproc to test because it requires Python 3
+        dataproc_image_version = '1.4-debian9'
```

Comment:
Just return if image_version less than 1.4, don't override it.

Look into other tests, some of them already doing similar thing.

Reply:
Does the integration test pinned at 1.3? https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/integration_tests/dataproc_test_case.py#L19

return it would result a successful test although it was not tested.

Reply:
Changed it to return when image version lower than 1.4